lib/rmg/RmgObject.cpp

```diff
@@ -271,6 +271,11 @@ const int3 & Object::getPosition() const
 int3 Object::getVisitablePosition() const
 {
 	assert(!dInstances.empty());
+
+	// FIXME: This doesn't take into account Hota monster offset
+	if (guarded)
+		return getGuardPos();
+
```

Comment:
It does, from what I see. Since `getGuardPos` returns visitable position, it should work with hota guards